### PR TITLE
OD-127 [Fix] Parse submit errors correctly

### PR DESCRIPTION
**Pull request by ivandevupp**
@sofiiakvasnevska
## Issue
OD-127 https://weboo.atlassian.net/browse/OD-127
## Description
Submit errors (from backend) are parsed correctly (not as cryptic and code-like).
## Screenshots/screencasts
https://monosnap.com/file/zexu8Dx5jcKnBe9XUpnnLcRuDsE175
## Backward compatibility
This change is fully backward compatible.
## Reviewers 
@upplabs-alex-levchenko @AndrRyaz

### CODE REVIEW

File: js/libs/form.js [hunk 1]
```diff
@@ -639,8 +639,7 @@ Fliplet.Widget.instance('form-builder', function(data) {
 
           $vm.loadEntryForUpdate();
         }, function(err) {
-          console.error(err);
```
Comment (squallstar) [inline js/libs/form.js:642]: why did you remove the `console.error`?

Comment (ivandevupp) [reply js/libs/form.js:642]: I took a look at other catch structures in widget, and found out that we don't use console there except one place in `image.js` which was added 4 years ago. Also I have a habit to clear consoles before delivering tasks.
If I it is useful for testing/Sentry I'll bring it back now.

Comment (squallstar) [reply js/libs/form.js:642]: Please bring it back, this is useful when clients and their developers are debugging via Fliplet Studio

Comment (ivandevupp) [reply js/libs/form.js:642]: Done.

**Review by squallstar — APPROVED** [review]
@sofiiakvasnevska this is ready to be deployed on **staging** for QA.

- **Risk**: Low
- **Affected**: Forms when displaying errors
- **Verify**: Verify errors are correctly displayed
